scripts/deploy.ui.js

```diff
@@ -49,7 +49,7 @@ class DeployUI extends BaseScript {
     }
     await remoteStorage.uploadDir(dist, this.config.s3.folder, f => this.emit('progress', path.basename(f)))
```

Comment:
`this.config.s3.folder` will need to be changed to include the namespace in the subfolder (once changes are pushed to the tvm as well)